students/BrianB/lesson03/slicing_lab.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+
+a_string = "this is a string"
+a_tuple = (2, 54, 13, 12, 5, 32)
+
+
+def exchange_first_last(seq):
+    """Returns an exchange of the first element with the last element."""
+    a_new_sequence = seq[-1:] + seq[1:-1] + seq[:1]
+    return a_new_sequence
+
+
+def every_other(seq):
+    """Returns every other element in the sequence."""
+    a_new_sequence = seq[0:-1:2]
```

Comment:
careful -- putting the -1 in there means you won't get the last one -- if there are an odd number of items, yo'll miss one.

and you don't need the 0 -- that's implied. So this can simply be: `[::2]`